feedback/sprint1-feedback.md

```diff
@@ -0,0 +1,64 @@
+# Feedback for alabaster-armadillo
+
+(X) tagged commit on main for sprint1
+(X) set of closed user stories
+(X) working deployment on Heroku
+(X) travis reports build passing
+(X) team members have completed reflection
+(X) demo
+
+## Checklist notes
+
+It would be good if you did a little more editing to your README so that readers would encounter your information first, rather than my generic discussion.
+
+## Discussion
+
+### Functionality
+
+I am glad that you got the lastFM integration in early. That is a good piece to be finished with so you can focus on the site itself. I also see that you got material-ui working, which is another good first step.
+
+What you showed this week is obviously more of a technology demo, which is appropriate given the time and where your focus was. The result, however, is a collection of buttons and entry fields that don't necessarily all make sense on the same page.
+
+For the next sprint I would like to see something that is more obviously a part of the final website. It would be good to sketch in the overall site -- remember that we are striving for incremental AND iterative progress.
+
+I would also encourage you to give some thought to the UX of your pages. For example, my understanding is that the DJs enter in their playlist _as they play the music_ so it tracks with what is playing. Make sure you support this. Another possible design might support the DJ entering a list of songs ahead of time and selecting them as she plays them. Talk with some folks who DJ and see what makes the most sense and then write a user story that encompasses that use case.
```

Comment:
This is actually not how it works. Currently, DJ's enter their playlists at the beginning of the show and there is no way of listeners to be able to track what is currently playing. The "currently playing" title is actually only the current radio show playing. I thinking talking about adding this feature might be nice though.

Reply:
For what it is worth, I spoke with Maddie, who assures me that DJs currently do enter in songs one at a time during the show. Not that you need to mimic that...